addons/stock/models/stock_move.py

```diff
@@ -127,6 +127,10 @@ def _default_group_id(self):
     propagate = fields.Boolean(
         'Propagate cancel and split', default=True,
         help='If checked, when this move is cancelled, cancel the linked move too')
+    propagate_date = fields.Boolean(string="Propagate Rescheduling", default=True,
```

Comment:
i don't think you should set the defaults to these field here, the value should be set by the stock rule and that's it